src/USER-MISC/fix_charge_regulation.h

```diff
@@ -0,0 +1,117 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Tine Curk (tcurk5@gmail.com) and Jiaxing Yuan (yuanjiaxing123@hotmail.com)
+------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+
+FixStyle(charge_regulation,Fix_charge_regulation)
```

Comment:
The name of this fix style should be `charge/regulation` and the name of the class should be `FixChargeRegulation` to be consistent with the conventions applied elsewhere in LAMMPS. This change needs - of course - be propagated to all relevant files in this pull request.